packages/workflow/src/internals.ts

```diff
@@ -619,10 +619,10 @@ export class Activator implements ActivationHandler {
   async handleWorkflowFailure(error: unknown): Promise<void> {
     if (this.cancelled && isCancellation(error)) {
       this.pushCommand({ cancelWorkflowExecution: {} }, true);
-    } else if (error instanceof ContinueAsNew) {
+    } else if (ContinueAsNew.is(error)) {
       this.pushCommand({ continueAsNewWorkflowExecution: error.command }, true);
     } else {
-      if (!(error instanceof TemporalFailure)) {
+      if (!TemporalFailure.is(error)) {
```

Comment:
@bergundy I added these two. Were they left apart intentionally?

Reply:
I must have missed this.